### DISCUSSION TITLE
Is there a pytest plugin to time all the individual tests and maybe rank them accordingly in the output.

### DISCUSSION THREAD

**Discussion by prady0t** [pytest-dev/pytest#12128] [Q&A]
I'm working on a project that plans to move from unittest to pytest. Since pytest supports unittest style tests out of the box, our first aim is to switch to pytest for CI jobs. But there are few tests that take really long time, it might be because how pytest is handeling those tests and we expect better test time once we have completely migrated to pytest. For the time being it would be really beneficial to have a ranking of tests that take highest amount of time so that we can start working on them beforehand.

Is there a way to achieve this?

**Comment by The-Compiler** [comment]
See https://docs.pytest.org/en/8.0.x/example/simple.html#profiling-test-duration

**Comment by prady0t** [comment]
Thanks a lot for the response!!
